webapp/components/abstract-test-file-results-table.html

```diff
@@ -8,19 +8,73 @@
   <script>
     // eslint-disable-next-line no-unused-vars
     const AbstractTestFileResultsTable = superClass => class extends superClass {
-      static get is() {
-        return 'abstract-test-file-results-table';
-      }
-
       static get properties() {
         return {
           subtestNames: {
             type: Array,
             value: [],
           },
+          isTestHarness: {
+            type: Boolean,
+            value: false,
+          },
         };
       }
 
+      static get observers() {
+        return ['fetchTestFile(path, testRuns)'];
+      }
+
+      async fetchTestFile(path, testRuns) {
+        if (!path || !testRuns) {
+          return;
+        }
+
+        const resultPerTestRun = await Promise.all(
+          testRuns.map(tr => this.loadResultFile(tr)));
+
+        let isTestHarness = false;
+        const subtestNames = ['STATUS'];
+        resultPerTestRun.forEach((resultData, i) => {
+          if (!resultData) {
+            testRuns[i].subtests = {};
+            testRuns[i].subtests['STATUS'] = { status: '(results not found)' };
+            return;
+          }
+          testRuns[i].subtests = {};
+          testRuns[i].subtests['STATUS'] = { status: resultData.status };
+          if (['OK', 'ERROR'].includes(resultData.status)) {
+            isTestHarness = true;
```

Comment:
I can't quite tell if the fix is around here and hard to spot because of the moved code, or if the refactoring itself fixes this as a side effect. Either works, but if there are hard-to-spot changes here, two commits would be appreciated for a nervous reviewer :)

Reply:
No, the real fix is the two new lines in `test-file-results.html`. I've split the changes into two commits.